### PR TITLE
Update the CRC list

### PR DESCRIPTION
**Pull request by TethysSvensson**

### CODE REVIEW

**Comment by zachriggle** [comment]
In order to reduce future diffs and make the important bits of the delta more important, can we run this through PEP8 or similar formatting?

**Review by zachriggle — APPROVED** [review]
LGTM

**Comment by TethysSvensson** [comment]
Done using autopep8

**Comment by zachriggle** [comment]
Merged via 5c72d62c9007d68b98e132ed9caa63afe6cb6805 (needed to rebase after 925cb430243fdf46067aeadbbdc924a9e6e279c2)

File: pwnlib/util/crc/known.py [hunk 1]
```diff
@@ -10,10 +10,11 @@ def generate(self):
     """
 
     curdir, _ = os.path.split(__file__)
-    path = os.path.join(curdir, '..', 'data', 'crcsums.txt')
+    path = os.path.join(curdir, '..', '..', 'data', 'crcsums.txt')
```
Comment (zachriggle) [inline pwnlib/util/crc/known.py:13]: Hey, what's this about, with changing the path?

**Pull request closed without merging**